util/envDynawo.sh

```diff
@@ -1552,18 +1609,34 @@ deploy_dynawo() {
     cp $DYNAWO_INSTALL_DIR/doxygen/Dynawo.tag doxygen/.
   fi
 
-  cp -r $DYNAWO_CURVES_TO_HTML_DIR sbin/.
-  mkdir -p sbin/nrt
-  cp -r $DYNAWO_HOME/util/nrt_diff sbin/nrt/.
+  mkdir -p sbin/curvesToHtml/csvToHtml sbin/curvesToHtml/resources sbin/curvesToHtml/xmlToHtml
+  cp -r $DYNAWO_CURVES_TO_HTML_DIR/*.py sbin/curvesToHtml/
```

Comment:
Same as above: harder to maintain

Reply:
It is to prevent temporary files copied as well

Reply:
you could also remove the .pyc before copying